receiptui/src/main/res/layout/fragment_receipt_detail.xml

```diff
@@ -43,7 +46,62 @@
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+                    <TextView android:id="@+id/transaction_title"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:text="@string/text_placeholder"
+                        style="@style/ReceiptTitle"
+                        android:textIsSelectable="true"
+                        android:focusable="true"
+                        android:longClickable="true"
+                        app:layout_constraintEnd_toStartOf="@+id/transaction_amount"
+                        app:layout_constraintStart_toEndOf="@+id/transaction_type_icon"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+                    <TextView
+                        android:id="@+id/transaction_date"
+                        style="@style/ReceiptDate"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:text="@string/text_placeholder"
+                        android:textIsSelectable="true"
+                        android:focusable="true"
+                        android:longClickable="true"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintEnd_toStartOf="@+id/transaction_currency"
+                        app:layout_constraintStart_toEndOf="@+id/transaction_type_icon"
+                        app:layout_constraintTop_toBottomOf="@+id/transaction_title" />
+
+                    <TextView android:id="@+id/transaction_amount"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/text_placeholder"
+                        style="@style/ReceiptAmount"
+                        android:textIsSelectable="true"
+                        android:focusable="true"
+                        android:longClickable="true"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+                    <TextView android:id="@+id/transaction_currency"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/text_placeholder"
+                        style="@style/ReceiptCurrency"
+                        android:textIsSelectable="true"
+                        android:focusable="true"
+                        android:longClickable="true"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintTop_toBottomOf="@+id/transaction_title" />
 
```

Comment:
why this section is added?

Reply:
<include layout="@layout/receipt" /> common for recent receipt, transaction list.

Reply:
can you explain why `<include layout="@layout/receipt" />` had to be deleted and replaced with 100 lines

Reply:
<include layout="@layout/receipt" /> is common for home recent recipe list and transaction recipe list ,while implement the text selection its reflect in home and transaction list.

Reply:
revert the `<include layout="@layout/receipt" />` text selection added in code